x/staging/posts/types/keys.go

```diff
@@ -84,9 +84,14 @@ func PostReactionsStoreKey(id string) []byte {
 	return append(PostReactionsStorePrefix, []byte(id)...)
 }
 
-// ReactionsStoreKey turns the combination of shortCode and subspace to a key used to store a reaction into the reaction's store
-func ReactionsStoreKey(shortCode, subspace string) []byte {
-	return append(ReactionsStorePrefix, []byte(shortCode+subspace)...)
+// ReactionSubspacePrefix returns the prefix used to store all the reactions for the subspace having the given id
+func ReactionSubspacePrefix(subspace string) []byte {
+	return append(ReactionsStorePrefix, []byte(subspace)...)
+}
+
+// ReactionsStoreKey turns the combination of subspace and shortCode to a key used to store a reaction into the reaction's store
+func ReactionsStoreKey(subspace, shortCode string) []byte {
```

Comment:
For the same reason above, I think we should rename this `RegisteredReactionStoreKey`